vip-application/src/main/java/fr/insalyon/creatis/vip/application/client/view/boutiquesParsing/BoutiquesParser.java

```diff
@@ -104,6 +111,15 @@ public BoutiquesApplication parseApplication(String descriptor) throws InvalidBo
                         exception);
             }
         }
+        // Custom property
+        JSONObject customObject = getObjectValue(parsedDescriptor, "custom", true);
+        if (customObject != null) {
+            try {
+                application.setVipContainer(getStringValue(customObject, "vip:imagepath", true));
+            } catch (InvalidBoutiquesDescriptorException exception) {
```

Comment:
i think we can avoid this exception catching, as the key is optional and the new exception does not add useful information to the original one.

Reply:
Done